consai_examples/consai_examples/field_observer.py

```diff
@@ -125,16 +122,12 @@ def _detection_tracked_callback(self, msg):
                 continue
             robot_id = robot.robot_id.id
             robot_is_yellow = robot.robot_id.team_color == RobotId.TEAM_COLOR_YELLOW
-            # TODO: ロボットの座標表現はリストではなくState2Dを使用する
             team_str = "our" if self._our_team_is_yellow == robot_is_yellow else "their"
             object_str = "self." + team_str + "_robots_"
-            eval(object_str + "pos")[robot_id] = [robot.pos.x, robot.pos.y]
-            eval(object_str + "angle")[robot_id] = robot.orientation
-
+            eval(object_str + "id").append(robot_id)
+            eval(object_str + "pos")[robot_id] = robot.pos
```

Comment:
`robot.pos`は`TrackedRobot`型で、`our_robots_pos`はState2D型です。

型が違うので修正してください

```suggestion
            eval(object_str + "pos")[robot_id] = State2D(x=robot.pos.x, y=robot.pos.y, theta=robot.orientation)
```

Reply:
メモ：書いた

Reply:
メモ：書いた